tests/integration/test_github_readonly.py

```diff
@@ -0,0 +1,46 @@
+import os
+
+import pytest
+from libpagure import APIError
+
+from ogr.abstract import PRStatus
+from ogr.services.mock.github import GithubService
+from tests.integration.conftest import skipif_not_all_env_vars_set
+
+pytestmark = skipif_not_all_env_vars_set(["GITHUB_TOKEN", "GITHUB_USER"])
```

Comment:
I hope we won't need tokens when using the mocked interface

Reply:
Hi, sure for `full mock` environment. But For read only mode, what will be implementing `--dry-run` for `packit`, it has to be able to read data (could be possible to use anonymous access to github, but it could be unreliable)